viewer/viewer.js

```diff
@@ -17,7 +17,7 @@
  */
 'use strict';
 
-const MIN_DB_VERSION = 62;
+const MIN_DB_VERSION = 65; // TODO ECR - need to up the version?
```

Comment:
@awick Do I need to up the db version here?

Reply:
Yes we probably do because the hunt_v2 is defined as static, if you try write a new field it will fail.